invenio_app_ils/records/resolver/jsonresolver/document_circulation.py

```diff
@@ -25,31 +25,35 @@ def jsonresolver_loader(url_map):
     def circulation_resolver(document_pid):
         """Return circulation info for the given Document."""
         item_search = ItemSearch()
-        loan_search = IlsLoansSearch()
+        items_count = item_search.search_by_document_pid(
+            document_pid).count()
 
-        all_items_count = item_search.search_by_document_pid(
-            document_pid).execute().hits.total
         unavailable_items_count = item_search. \
-            get_unavailable_items_by_document_pid(
-                document_pid).execute().hits.total
+            get_unavailable_items_by_document_pid(document_pid).count()
 
+        loan_search = IlsLoansSearch()
         past_loans_count = loan_search.get_past_loans_by_doc_pid(
-            document_pid).execute().hits.total
+            document_pid).count()
         active_loans_count = loan_search.get_active_loans_by_doc_pid(
-            document_pid).execute().hits.total
+            document_pid).count()
         pending_loans_count = loan_search.get_pending_loans_by_doc_pid(
-            document_pid).execute().hits.total
+            document_pid).count()
 
-        items_available_for_loan = all_items_count - \
+        eitem_search = EItemSearch()
+        eitems_count = eitem_search.search_by_document_pid(
+            document_pid).count()
+
+        items_available_for_loan = items_count - \
             active_loans_count - unavailable_items_count
 
         circulation = {
-            "number_of_past_loans": past_loans_count,
-            "number_of_items": all_items_count,
             "active_loans": active_loans_count,
             "items_available_for_loan": items_available_for_loan,
-            "pending_loans": pending_loans_count,
+            "number_of_eitems": eitems_count,
```

Comment:
I am not sure I would put `number_of_eitems` and `number_of_items` inside the `circulation` object, it is not really related.

Reply:
It might get messy if we keep adding new `_` properties to the schema. We could store `stats` in `_computed` to keep it a bit more organized.